PubSub/src/Topic.php

```diff
@@ -182,6 +182,96 @@ public function create(array $options = [])
         return $this->info;
     }
 
+    /**
+     * Update the topic.
+     *
+     * Note that the topic's name and kms key name are immutable properties and may not be modified.
+     *
+     * Example:
+     * ```
+     * $topic->update([
+     *     'messageStoragePolicy' => [
+     *         'allowedPersistenceRegions' => ['us-central1']
+     *     ]
+     * ]);
+     * ```
+     *
+     * ```
+     * // Updating labels with an explicit update mask
+     * $topic->update([
+     *     'labels' => [
+     *         'foo' => 'bar'
+     *     ]
+     * ], [
+     *     'updateMask' => [
+     *         'labels'
+     *     ]
+     * ]);
+     * ```
+     *
+     * @param array $topic {
+     *     The Topic data.
+     *
+     *    @type array $labels Key value pairs used to organize your resources.
+     *    @type string $kmsKeyName The resource name of the Cloud KMS
```

Comment:
If this is immutable, why document it here?

Reply:
Good catch, removed!